CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 1.3.0
+
+This is a "feature" and security release.
+
+* [TK-178](https://tickets.puppetlabs.com/browse/TK-178) Upgrade to version
+  v9.2.10.  This includes changes made in the v9.2.9 to address a critical
```

Comment:
Maybe it's obvious, but we may want to explicitly state we upgraded to "Jetty v9.2.10"